### PR TITLE
add fn to make path from prot_id

### PR DESCRIPTION
**Pull request by BobBorges**
...used in the quality estimation tests

### CODE REVIEW

File: pyriksdagen/utils.py [hunk 1]
```diff
@@ -516,3 +516,29 @@ def get_context_sequences_for_protocol(protocol, context_type, target_length = 1
...
     return output_dict
+
+
+def pathize_protocol_id(protocol_id):
+    """
+    Turn the protocol id into a path string from riksdagen-records root. Handles 'unpadded' protocol numbers.
+    """
+
+    spl = protocol_id.split('-')
+    py = spl[1]
```
Comment (ninpnin) [inline pyriksdagen/utils.py:527]: I would write out the abbreviations for maintainability

Comment (BobBorges) [reply pyriksdagen/utils.py]: [8006e61](https://github.com/swerik-project/pyriksdagen/pull/18/commits/8006e612e5069ba9fd4557cc42369f6fe1eaacaa)

File: pyriksdagen/utils.py [hunk 2]
```diff
@@ -516,3 +516,29 @@ def get_context_sequences_for_protocol(protocol, context_type, target_length = 1
...
+    else:
+        nr = spl[5]
+        pren = '-'.join(spl[:5])
+        if len(spl) == 7:
+            suffix = f"-{spl[-1]}"
+    path_ = f"data/{py}/{pren}-{nr:0>3}{suffix}.xml"
+    if os.path.exists(path_):
+        return path_
+    else:
+        path_ = re.sub(f'((extra)?h[^-]+st|")', '', path_)
```
[the remark this answers is not in the document]
Comment (BobBorges) [reply pyriksdagen/utils.py:545]: [e180bc4](https://github.com/swerik-project/pyriksdagen/pull/18/commits/e180bc402ea1b491fbb856ada914cc8773a48574)

**Review by ninpnin — APPROVED** [review]
LGTM

**Pull request merged**